### PR TITLE
Update welcome and readme

### PR DESCRIPTION
**Pull request by daniels290813**
https://iguazio.atlassian.net/browse/IG-23192

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,13 +106,27 @@ For full usage instructions, run the script with the `-h` or `--help` flag:
...
-    <tr>
+     <tr>
+    <td style='min-width:45px;'>
+        <b><Call center</b>
+        </td>
+		<td align="center", style="min-width:45px; padding: 10px;">
+            <a target="_blank" href="https://github.com/mlrun/demo-call-center/blob/main/">
+                <img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
+        </td>
+        <td>This demo showcases how to use LLMs to turn audio files, from call center conversations between
```
Comment (jillnogold) [inline README.md]: <td>This demo showcases how to use LLMs to turn audio files from call center conversations, between 
		customers and agents, into valuable data &mdash; all in a single workflow orchestrated by MLRun.
        MLRun automates the entire workflow, auto-scales resources as needed, and automatically logs and 
		parses values between the different workflow steps.

**Comment by daniels290813** [comment]
Thanks @jillnogold ! can i merge ?

File: README.md [hunk 2]
```diff
@@ -95,13 +106,27 @@ For full usage instructions, run the script with the `-h` or `--help` flag:
...
+        </td>
+    </tr>	
+	<tr>
         <td><b>Mask detection</b></td>
         <td align="center", style="min-width:45px; padding: 10px;">
             <a href="demos/mask-detection/README.md"><img src="./assets/images/Jupyter-Logo-32px.png"/><br>Open locally</a>
         </td>
         <td align="center", style="min-width:45px; padding: 10px;">
-            <a target="_blank" href="https://github.com/mlrun/demos/tree/master/mask-detection/">
+            <a target="_blank" href="https://github.com/mlrun/demos/tree/release/v1.6.x-latest/mask-detection/">
```
Comment (jillnogold) [inline README.md:129]: Shouldn't these be the 1.7.x versions?
add https://github.com/mlrun/demo-sagemaker

File: README.md [hunk 3]
```diff
@@ -128,7 +153,7 @@ For full usage instructions, run the script with the `-h` or `--help` flag:
             <a href="demos/news-article-nlp/README.md"><img src="./assets/images/Jupyter-Logo-32px.png"/><br>Open locally</a>
         </td>
         <td align="center", style="min-width:45px; padding: 10px;">
-            <a target="_blank" href="https://github.com/mlrun/demos/tree/master/news-article-nlp/"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
+            <a target="_blank" href="https://github.com/mlrun/demos/tree/release/v1.6.x-latest/news-article-nlp/"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
```
Comment (jillnogold) [inline README.md:156]: deprecated

File: README.md [hunk 4]
```diff
@@ -140,7 +165,7 @@ Additionally, we will use MLRun's real-time inference graphs to create the pipel
             <a href="demos/network-operations/README.md"><img src="./assets/images/Jupyter-Logo-32px.png"/><br>Open locally</a>
         </td>
         <td align="center", style="min-width:45px; padding: 10px;">
-            <a target="_blank" href="https://github.com/mlrun/demos/tree/master/network-operations/"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
+            <a target="_blank" href="https://github.com/mlrun/demos/tree/release/v1.6.x-latest/network-operations/"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
```
Comment (jillnogold) [inline README.md:168]: deprecated

File: README.md [hunk 5]
```diff
@@ -179,7 +204,7 @@ The demo uses a offline/real-time metrics simulator to generate semi-random netw
             <a href="demos/howto/converting-to-mlrun/mlrun-code.ipynb"><img src="./assets/images/Jupyter-Logo-32px.png"/><br>Open locally</a>
         </td>
         <td align="center", style="min-width:45px; padding: 10px;">
-            <a target="_blank" href="https://github.com/mlrun/demos/tree/master/howto/converting-to-mlrun"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
+            <a target="_blank" href="https://github.com/mlrun/demos/tree/release/v1.6.x-latest/master/howto/converting-to-mlrun"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
```
Comment (jillnogold) [inline README.md:207]: deprecated

File: README.md [hunk 6]
```diff
@@ -191,7 +216,7 @@ The demo uses a offline/real-time metrics simulator to generate semi-random netw
             <a href="demos/howto/spark/spark-mlrun-read-csv.ipynb"><img src="./assets/images/Jupyter-Logo-32px.png"/><br>Open locally</a>
         </td>
         <td align="center", style="min-width:45px; padding: 10px;">
-            <a target="_blank" href="https://github.com/mlrun/demos/tree/master/howto/spark/spark-mlrun-read-csv.ipynb"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
+            <a target="_blank" href="https://github.com/mlrun/demos/tree/release/v1.6.x-latest/howto/spark/spark-mlrun-read-csv.ipynb"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
```
Comment (jillnogold) [inline README.md:219]: deprecated

File: README.md [hunk 7]
```diff
@@ -202,7 +227,7 @@ The demo uses a offline/real-time metrics simulator to generate semi-random netw
             <a href="demos/howto/spark/spark-mlrun-describe.ipynb"><img src="./assets/images/Jupyter-Logo-32px.png"/><br>Open locally</a>
         </td>
         <td align="center", style="min-width:45px; padding: 10px;">
-            <a target="_blank" href="https://github.com/mlrun/demos/tree/master/howto/spark/spark-mlrun-describe.ipynb"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
+            <a target="_blank" href="https://github.com/mlrun/demos/tree/release/v1.6.x-latest/howto/spark/spark-mlrun-describe.ipynb"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
```
Comment (jillnogold) [inline README.md:230]: deprecated

File: README.md [hunk 8]
```diff
@@ -213,7 +238,7 @@ The demo uses a offline/real-time metrics simulator to generate semi-random netw
             <a href="demos/howto/spark/spark-operator.ipynb"><img src="./assets/images/Jupyter-Logo-32px.png"/><br>Open locally</a>
         </td>
         <td align="center", style="min-width:45px; padding: 10px;">
-            <a target="_blank" href="https://github.com/mlrun/demos/tree/master/howto/spark/spark-operator.ipynb"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
+            <a target="_blank" href="https://github.com/mlrun/demos/tree/release/v1.6.x-latest/howto/spark/spark-operator.ipynb"><img src="./assets/images/GitHub-Mark-32px.png"/><br>View on GitHub</a>
```
Comment (jillnogold) [inline README.md:241]: deprecated

**Pull request closed without merging**